tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -744,18 +744,22 @@ virtual void TearDown() {
 
 TEST(CTFont, GetVerticalTranslationsForGLyphs) {
     auto font = woc::MakeAutoCF<CTFontRef>(CTFontCreateWithName(CFSTR("Arial"), 20, nullptr));
-    UniChar chars[4] = { 'T', 'e', 's', 't' };
-    CGGlyph glyphs[4];
-    CTFontGetGlyphsForCharacters(font, chars, glyphs, 4);
-
-    CGSize translations[4];
-    CTFontGetVerticalTranslationsForGlyphs(font, glyphs, translations, 4);
-    EXPECT_NEAR(-6.108, translations[0].width, c_errorMargin);
-    EXPECT_NEAR(-5.562, translations[1].width, c_errorMargin);
+    UniChar chars[6] = { 'T', 'e', 's', 't', ' ', '_' };
```

Comment:
vaguely interested in '-' and '|' also, but we probably won't find anything surprising, so i'll leave it up to you